### PR TITLE
Fit the new storage queyr interface.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Follow https://github.com/vesoft-inc/nebula-common/pull/323
Follow https://github.com/vesoft-inc/nebula-storage/pull/224

### CODE REVIEW

**Review by CPWstatic — APPROVED** [review]
LGTM.

**Review by nevermore3 — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

File: src/validator/IndexScanValidator.cpp [hunk 1]
```diff
@@ -46,6 +58,18 @@ Status IndexScanValidator::prepareFrom() {
...
+        returnCols_->emplace_back(kSrc);
+        colNames_.emplace_back(kSrcVID);
+        returnCols_->emplace_back(kDst);
+        colNames_.emplace_back(kDstVID);
+        returnCols_->emplace_back(kRank);
+        colNames_.emplace_back(kRanking);
+    } else {
+        returnCols_->emplace_back(kVid);
+        colNames_.emplace_back(kVertexID);
+    }
```
Comment (bright-starry-sky) [inline src/validator/IndexScanValidator.cpp:72]: Why save two copies? IIRR，Other PR is already dealing with this issue.

Comment (Shylock-Hg) [reply src/validator/IndexScanValidator.cpp:72]: They are different see the line 112

File: src/planner/match/PropIndexSeek.cpp [hunk 2]
```diff
@@ -60,19 +61,22 @@ StatusOr<SubPlan> PropIndexSeek::transformNode(NodeContext* nodeCtx) {
...
     auto columns = std::make_unique<std::vector<std::string>>();
+    columns->emplace_back(kVid);
     auto scan = IndexScan::make(matchClauseCtx->qctx,
                                 nullptr,
                                 matchClauseCtx->space.id,
                                 std::move(contexts),
                                 std::move(columns),
                                 false,
                                 nodeCtx->scanInfo.schemaId);
+    scan->setColNames({kVid});
```
Comment (bright-starry-sky) [inline src/planner/match/PropIndexSeek.cpp:72]: What is the purpose here, and why is it not the same as below?
```
returnCols_->emplace_back(kVid);
colNames_.emplace_back(kVertexID);
```

Comment (Shylock-Hg) [reply src/planner/match/PropIndexSeek.cpp:72]: Let user reference result of lookup.

File: src/validator/IndexScanValidator.cpp (hunk 1 above)
Comment (bright-starry-sky) [reply src/validator/IndexScanValidator.cpp:72]: > They are different

Yes, I see a difference, but I think one copy is enough

File: src/planner/match/PropIndexSeek.cpp (hunk 2 above)
Comment (Shylock-Hg) [reply src/planner/match/PropIndexSeek.cpp:72]: E.G. `lookup ... | FETCH ON PROP $-.VertexID`.

Comment (Shylock-Hg) [reply src/planner/match/PropIndexSeek.cpp:72]: And make the column names same as 1.0

Comment (bright-starry-sky) [reply src/planner/match/PropIndexSeek.cpp:72]: > Let user reference result of lookup.

Why  different with `IndexScanValidator::prepareYield`?

File: src/validator/IndexScanValidator.cpp (hunk 1 above)
Comment (bright-starry-sky) [reply src/validator/IndexScanValidator.cpp:72]: Is there conflict with other PR : 
https://github.com/vesoft-inc/nebula-storage/pull/224

File: src/planner/match/PropIndexSeek.cpp (hunk 2 above)
Comment (Shylock-Hg) [reply src/planner/match/PropIndexSeek.cpp:72]: The prepareYield won't affect the column names of result, It's a bug I will fix in latter PR.

File: src/validator/IndexScanValidator.cpp (hunk 1 above)
Comment (Shylock-Hg) [reply src/validator/IndexScanValidator.cpp:72]: Won't I test in this PR.

Comment (bright-starry-sky) [reply src/validator/IndexScanValidator.cpp:72]: > Won't I test in this PR.

got.

**Review by CPWstatic — APPROVED** [review]

**Pull request merged**